cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,104 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
+
+const SAFE_TEST = '0x8675B754342754A30A2AeF474D114d8460bca19b'
```

Comment:
`TEST_SAFE`?

And please include the network prefix in the address.

Reply:
done